docs/helpers/ListViewBaseExtensions.md

```diff
@@ -0,0 +1,29 @@
+# ListViewBaseExtensions class
+
+ListViewBaseExtensions provides extension method that allow attaching ICommand to handle ListViewBase Item interaction by means of [ItemClick](https://msdn.microsoft.com/en-us/library/windows/apps/windows.ui.xaml.controls.listviewbase.itemclick.aspx) event. 
+ListViewBase [IsItemClickEnabled](https://msdn.microsoft.com/en-us/library/windows/apps/windows.ui.xaml.controls.listviewbase.isitemclickenabled.aspx) must be set to **true**
+
+
+
+## Example
+
+```xaml
```

Comment:
There is no `xaml` highlighting support, and in essence, this is `xml`, so I suggest using that instead!